### DISCUSSION TITLE
Notify of new release available when running `dab --version`

### DISCUSSION THREAD

**Discussion by yorek** [Azure/data-api-builder#1366] [Ideas]
That would help to know when a new release is available, without having to check for GitHub page

**Comment by abhishekkumams** [comment]
There is an [issue](https://github.com/NuGet/Home/issues/3292) already open for the same in the Nuget repo, where users wants to get notified for any update.

While, dotnet has a way to check the latest version without going to Github page:
```bash
dotnet tool search Microsoft.DataApiBuilder
```
This will output the latest released version available for downloads.

![image](https://user-images.githubusercontent.com/102276754/229453293-5ab1cdeb-eb53-4e72-a50f-35db2a983ef2.png)
